File: C/1. Introduction/exercise4.c
// Pointers in C "https://www.hackerrank.com/challenges/pointer-in-c/problem"

#include <stdio.h>
#include <stdlib.h>

void update(int *a,int *b) {
    // Complete this function
    // Creating a temporary variable because in the next line a* will change so getting abs will not work
    int temp_a = *a;
    *a += *b;
    *b = abs(temp_a - *b);
}

int main() {
    int a, b;
    int *pa = &a, *pb = &b;

    scanf("%d %d", &a, &b);
    update(pa, pb);
    printf("%d\n%d", a, b);

    return 0;
}

File: Java/1. Introduction/exercise4.java
// Java Output Formatting"https://www.hackerrank.com/challenges/java-output-formatting/problem"

import java.util.Scanner;


public class Exercise4 {

    public static void main(String[] args) {
            Scanner sc = new Scanner(System.in);
            System.out.println("================================");
            for(int i=0; i< 3; i++){
                String s1 = sc.next();
                int x = sc.nextInt();
                //Complete this line
                System.out.printf("%-15s%03d%n", s1, x);
            }
            System.out.println("================================");
            sc.close();
    }
}
